src/modules/entities/modules/reports/services/sagas/fetchReports.js

```diff
@@ -45,13 +47,25 @@ import { fetchReports as actions, fetchReportsTypes } from '../actions';
 
 function* fetchReports() {
     try {
-        const snapshot = yield firestore.collection('messages').limit(1).get();
-
+        const snapshot = yield firestore.collection('messages').limit(5).get();
         const data = [];
+        const results = [];
+        let id = 0;
```

Comment:
move this down above the iterating the snapshot for images to be more connected to the code there